### PR TITLE
Feat/resend tx

### PR DESCRIPTION
**Pull request by JDonadio**

### CODE REVIEW

File: src/js/controllers/walletHome.js [hunk 1]
```diff
@@ -1098,6 +1098,7 @@ angular.module('copayApp.controllers').controller('walletHomeController', functi
       $scope.btx = btx;
       $scope.settings = walletSettings;
       $scope.color = fc.backgroundColor;
+      $scope.network = fc.credentials.network;
```
Comment (cmgustavo) [inline src/js/controllers/walletHome.js]: Duplicated. This variable could be get by `getShortNetworkName()`

File: public/views/modals/tx-details.html [hunk 2]
```diff
@@ -146,6 +146,9 @@ <h4 class="title m0" translate>Participants</h4>
         (getShortNetworkName() == 'test' ? 'test-' : '') + 'insight.bitpay.com/tx/' + btx.txid)">
         <span class="text-gray" translate>See it on the blockchain</span>
       </button>
+      <button class="button outline round dark-gray tiny" ng-show="btx.action == 'sent' && network == 'testnet'" ng-click="sendAgain(btx)">
```
Comment (cmgustavo) [inline public/views/modals/tx-details.html]: The style of button should be different than the above.

**Comment by tanojaja** [comment]
Updated. Thanks for comment!

File: public/views/modals/tx-details.html [hunk 3]
```diff
@@ -146,6 +146,11 @@ <h4 class="title m0" translate>Participants</h4>
         (getShortNetworkName() == 'test' ? 'test-' : '') + 'insight.bitpay.com/tx/' + btx.txid)">
         <span class="text-gray" translate>See it on the blockchain</span>
       </button>
+      <ul class="no-bullet m0" ng-show="btx.action == 'sent' && getShortNetworkName() == 'test'">
+        <li ng-click="sendAgain(btx)">
+          <span ng-style="{'color':color}" translate>Send again</span>
```
Comment (matiu) [inline public/views/modals/tx-details.html]: text should be "Send similar transaction"

File: public/views/modals/tx-details.html [hunk 4]
```diff
@@ -146,6 +146,11 @@ <h4 class="title m0" translate>Participants</h4>
         (getShortNetworkName() == 'test' ? 'test-' : '') + 'insight.bitpay.com/tx/' + btx.txid)">
         <span class="text-gray" translate>See it on the blockchain</span>
       </button>
+      <ul class="no-bullet m0" ng-show="btx.action == 'sent' && getShortNetworkName() == 'test'">
```
Comment (gabrielbazan7) [inline public/views/modals/tx-details.html]: Background color should be grey.

**Comment by matiu** [comment]
needs rebase.

**Comment by JDonadio** [comment]
No longer works with master branch

**Pull request closed without merging**